code/modules/SCP/SCPs/SCP-049.dm

```diff
@@ -23,9 +31,6 @@ GLOBAL_LIST_EMPTY(scp049_1s)
 	designation = "049"
 	classification = EUCLID
 
-/mob/living/carbon/human/scp049/IsAdvancedToolUser()
-	return FALSE
```

Comment:
Why this exacly?

Reply:
It feels unneeded given how skills work. He should be able to use guns/tools even if poorly.

Reply:
> Makes them lack dexterity, zombies shouldn't be using tools

This is 049 not 049-1.